### DISCUSSION TITLE
Is it possible to embed code from another file?

### DISCUSSION THREAD

**Discussion by Ravencentric** [retypeapp/retype#657] [Q&A] [answered]
for example, I have `docs/static/python/world.py` as such:
```py
print("hello world")
print("hello again")
```

Can I embed the code from this file somewhere else? An example:

code:
`````
```python:docs/static/python/world.py
 
```
`````
preview:
```py
print("hello world")
print("hello again")
```

Possibly with line selector too?

code:
`````
```python:docs/static/python/world.py[1:2]
 
```
`````
preview:
```py
print("hello world")
```

Something like so I can have multiple pages reference the same file without having to maintain multiple copies.

**Comment by DannyBen** [comment, marked as answer]
[Code SNippet](https://retype.com/components/code-snippet/#code-snippet)

**Reply by Ravencentric** [reply]
I missed it because I was only looking into the code blocks section. Thank you
